BFS.py

```diff
@@ -3,8 +3,6 @@
 import queue as q
 import board as b
 import pprint
-np.set_printoptions(edgeitems=30, linewidth=100000,formatter=dict(float=lambda x: "%.3g" % x))
```

Comment:
are these print options specific to the BFS class only?  Or do we want this in the main.py file?

Reply:
just for nice output of board when testing